src/main/java/games/strategy/engine/framework/headlessGameServer/AvailableGames.java

```diff
@@ -34,35 +34,11 @@
   private static final Logger logger = Logger.getLogger(AvailableGames.class.getName());
 
   private static final String ZIP_EXTENSION = ".zip";
-  private final Map<String, URI> availableGames = new TreeMap<>();
-  private final Set<String> availableMapFolderOrZipNames = new HashSet<>();
+  private final Map<String, URI> availableGames = Collections.synchronizedMap(new TreeMap<>());
+  private final Set<String> availableMapFolderOrZipNames = Collections.synchronizedSet(new HashSet<>());
```

Comment:
I recall having a discussion with @ssoloff about this, because the only occurence where synchronization is needed is during the parallel loading of the maps.

Reply:
It's not quite existing logic.  The synchronization only occurred while the (static) `populateAvailableGames()` method was running because the collections are accessed by multiple threads loading games in parallel.  After that point, the collections are effectively immutable and no longer need to be synchronized.

This probably could have been clearer if `populateAvailableGames()` returned the collections it initialized in a `Tuple` or some other container-like class, the components of which are then assigned to the class fields.